ci/run.sh

```diff
@@ -153,6 +153,52 @@ function gg_sum_ctest_release {
     gg_printf '```\n'
 }
 
+# test_scripts_debug
+
+function gg_run_test_scripts_debug {
+    cd ${SRC}
+
+    set -e
+
+    (find ./examples -mindepth 2 -maxdepth 2 -name 'tests.sh' -execdir time bash "{}" "$SRC/build-ci-debug/bin" "$MNT/models" \;) 2>&1 | tee -a $OUT/${ci}-scripts.log
+
+    set +e
+}
+
+function gg_sum_test_scripts_debug {
+    gg_printf '### %s\n\n' "${ci}"
+
+    gg_printf 'Runs test scripts in debug mode\n'
+    gg_printf '- status: %s\n' "$(cat $OUT/${ci}.exit)"
+    gg_printf '```\n'
+    gg_printf '%s\n' "$(cat $OUT/${ci}-scripts.log)"
+    gg_printf '```\n'
+    gg_printf '\n'
+}
+
+# test_scripts_release
+
+function gg_run_test_scripts_release {
+    cd ${SRC}
+
+    set -e
+
+    (find ./examples -mindepth 2 -maxdepth 2 -name 'tests.sh' -execdir time bash "{}" "$SRC/build-ci-release/bin" "$MNT/models" \;) 2>&1 | tee -a $OUT/${ci}-scripts.log
```

Comment:
Let's list explicitly the scripts that we run here - it's more clear what is being executed and easier to turn on/off certain tests

Reply:
Done